LayoutKitSampleApp/UrlImageLayout.swift

```diff
@@ -16,21 +16,28 @@ import LayoutKit
 class UrlImageLayout: SizeLayout<UrlImageView> {
 
     init(url: NSURL, size: CGSize) {
-        super.init(width: size.width, height: size.height, alignment: .center, flexibility: .inflexible, config: { imageView in
-            imageView.backgroundColor = UIColor.orangeColor()
-            imageView.url = url
-            dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), {
-                guard let data = NSData(contentsOfURL: url) else {
-                    NSLog("failed to load image data \(url)")
-                    return
-                }
-                dispatch_async(dispatch_get_main_queue(), {
-                    if imageView.url == url {
-                        imageView.image = UIImage(data: data)
-                    }
-                })
-            })
-        })
+        super.init(minWidth: size.width,
```

Comment:
consider adding a newline after the ( to reduce the indentation on the rest of the lines